manifests/init.pp

```diff
@@ -117,6 +119,7 @@
   $keepalive_timeout                                         = '65s',
   $keepalive_requests                                        = '100',
   $log_format                                                = {},
+  $stream_log_format                                         = {},
```

Comment:
Please enforce the datatype here as well. I guess something like this should work:
```suggestion
  Hash[String[1], String[1]] $stream_log_format              = {},
```